srcpkgs/radicale/template

```diff
@@ -1,20 +1,21 @@
 # Template file for 'radicale'
 pkgname=radicale
-version=1.1.6
-revision=4
+version=3.0.3
+revision=1
 archs=noarch
 wrksrc="Radicale-${version}"
 build_style=python3-module
-pycompile_module="radicale"
 hostmakedepends="python3 python3-setuptools"
-depends="python3-pam python3-requests python3-dulwich"
+depends="python3-pam python3-requests python3-dulwich python3-vobject
+ python3-dateutil python3-passlib python3-bcrypt python3-defusedxml
+ apache-htpasswd"
 short_desc="Complete calendar and contact storing and manipulating solution"
-maintainer="lemmi <lemmi@nerd2nerd.org>"
+maintainer="projectmoon <projectmoon@agnos.is>"
```

Comment:
Did @lemmi ask you to take over this template?

Reply:
No. This was copied from the template for radicale3. Do you want me to change the maintainer back?

Reply:
Just saw his comment.